conda/lock-environment.sh

```diff
@@ -35,4 +36,4 @@ conda-lock render -k env
 if [[ $? -eq 0 ]]; then
   conda deactivate
   conda remove -q -y --name ${LOCK_ENV} --all
-fi
+fi
```

Comment:
```suggestion
fi

```

Reply:
are the blank newlines important for running?

Reply:
Not really, it's just good to be consistent (have all files ending with a newline) for diff-ing. See https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file.

![image](https://user-images.githubusercontent.com/23487320/196501988-e0481fb8-9e60-4f66-b085-e9e2db11507c.png)